ovos_phal_connectivity_events/__init__.py

```diff
@@ -38,37 +39,52 @@ def __init__(self, bus=None, config=None):
         self.sleep_time = 60
         self.state = ConnectivityState.UNKNOWN
         self.bus.on("ovos.PHAL.internet_check", self.handle_check)
```

Comment:
this should be refactored now that plugins are Thread subclasses, should just be moved into `run` method instead of event based